ocaml-lsp-server/src/doc_to_md.ml

```diff
@@ -76,6 +76,13 @@ and inline_element_list_to_inlines inlines =
   let inlines = List.map ~f:inline_element_to_inline inlines in
   Inline.Inlines (inlines, Meta.none)
 
+(** Hack: Because of bug described here:
+    https://github.com/dbuenzli/cmarkit/issues/7 , we use only first block line.
+    It contains whole string, which seems good enough for our use cases. To be
+    removed when: https://github.com/dbuenzli/cmarkit/pull/8 gets merged and
+    released *)
+let block_lines_of_string str = [ List.hd (Block_line.list_of_string str) ]
```

Comment:
But then that's simply `Fun.id` with the bug no ?

Reply:
We used cmarkit 0.2.0 in the end (thanks for the release!), so we removed that workaround.